lib/dialects/cockroachdb.ts

```diff
@@ -0,0 +1,466 @@
+import { Knex } from 'knex';
+import { SchemaInspector } from '../types/schema-inspector';
+import { Table } from '../types/table';
+import { Column } from '../types/column';
+import { ForeignKey } from '../types/foreign-key';
+import { stripQuotes } from '../utils/strip-quotes';
+import isNil from 'lodash.isnil';
+
```

Comment:
It doesn't share anything with PG? Wonder if it could extend the base class or import some of the extracted code.

Reply:
updated it now to import everything that was not changed from postgres